src/development/coding-securely-personal-data.md

```diff
@@ -0,0 +1,218 @@
+
+Regulations require us to understand when personal data is being captured, to identify the risks to the person whose data it is, and to take appropriate steps to secure the data. The Special Categories of personal data defined in this section are always considered high risk.
+
+### Direct identifiers
+
+A direct identifier is a piece of data that by itself will identify someone.
+
+* General identifiers e.g. NHS Number or National Insurance Number
+* Biometric information e.g. fingerprint, face, Iris etc.
+* Local identifiers e.g. Pension number, employment number, login id, Pseudonymised identifiers etc.
+* Email address
+* Home phone number
+* Mobile phone or device number
+* Online identifiers e.g. IP address, event logs
+
+### Indirect Identifiers
+
+You need to use the information in combination with the information below or other information.  If in doubt please contact Information Governance.
+
+This list is meant to be helpful but not comprehensive.
+
+* Name
+* Address
+* Postcode (it can be a direct identifier if in a rural area with a low population density)
+* Date of Birth
+* Age
+* Sex
+* Marital Status
+* Gender
+* Living habits e.g. recorded by a smart meter or internet of things devices
+* Professional training/awards
+* Income/financial/Tax situation
+* Physical description
+* Website Cookies or technologies that achieve the same result e.g. Beacons
+* Location Data (Travel/GPS/GSM data)
+* Device MAC address (Wireless interface)
+* Singled out in statistical information
+* Unique mannerisms, ways or writing or unique ways of talking
+* The way you walk (everyone is unique)
+* Education/ professional training
+* Employment / Career history
+* Financial affairs
+* Signature
+* Genetic data – can be personal data for more than one person
+* Offences committed / Alleged to have been committed
+* Criminal proceedings/outcomes/sentences
+* Physical /mental Health or Condition
+* Sexual Life
+* Sexual Orientation
+* Family / Lifestyle/ Social Circumstances e.g. a social care record
+* Religion or other beliefs
+* Trade union membership
+* Racial / Ethnic origin
+
+### What is not Personal Data
+
+* Summary statistical information with values above an agreed value e.g. 5
+* Personal data that has been anonymised to the NHSBSA Anonymisation standard
+* Information that does not relate to a person e.g. is about an inanimate object / concept and no individual can be linked to it e.g. by ownership or use
+* Information about the deceased.  This can remain confidential e.g. Prescription and dental data or medical reports.
+
+### Special Categories of Personal data
+
+The use of the following types of personal data is prohibited unless permitted by law (GDPR Article 9):
+
+* Physical /mental Health or Condition
+* Sexual Life
+* Sexual Orientation
+* Family / Lifestyle/ Social Circumstances e.g. a social care record
```

Comment:
`Family / Lifestyle/ Social Circumstances` should be ` Family / Lifestyle / Social Circumstances`

Reply:
fixed to lowercase